fluka_vmc.sh

```diff
@@ -1,10 +1,11 @@
 package: FLUKA_VMC
 version: "%(tag_basename)s"
-tag: "4-1.1-vmc1"
+tag: "4-1.1-vmc2"
 source: https://gitlab.cern.ch/ALICEPrivateExternals/FLUKA_VMC.git
 requires:
   - "GCC-Toolchain:(?!osx)"
   - ROOT
```

Comment:
We can remove ROOT as VMC already depends on ROOT

Reply:
removed

Reply:
It turns out that this is required when we build FairRoot without simulation; can you put ROOT back in fairroot.sh?